src/bin/nickel.rs

```diff
@@ -16,68 +16,66 @@ use std::{
 };
 // use std::ffi::OsStr;
 use directories::BaseDirs;
-use structopt::StructOpt;
 
 /// Command-line options and subcommands.
-#[derive(StructOpt, Debug)]
+#[derive(clap::Parser, Debug)]
 /// The interpreter of the Nickel language.
 struct Opt {
     /// The input file. Standard input by default
-    #[structopt(short = "f", long, global = true, parse(from_os_str))]
+    #[arg(short, long, global = true)]
     file: Option<PathBuf>,
 
     #[cfg(debug_assertions)]
     /// Skips the standard library import. For debugging only. This does not affect REPL
-    #[structopt(long)]
+    #[arg(long)]
     nostdlib: bool,
 
     /// Coloring: auto, always, never.
-    #[structopt(long, global = true, case_insensitive = true, default_value = "auto")]
-    color: ColorOpt,
+    #[arg(long, global = true, value_enum, default_value_t = clap::ColorChoice::default())]
```

Comment:
According to [the Clap documentation](https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes), you should be able to omit the value when it is equal to `OptionType::default()`

```suggestion
    #[arg(long, global = true, value_enum, default_value_t)]
```

Reply:
Great catch! Fixed in 5b8919a1